modules/library/main/src/main/java/org/geotools/geometry/jts/WKBReader.java

```diff
@@ -259,18 +259,18 @@ private Geometry setSRID(Geometry g, int SRID) {
         return g;
     }
 
-    private Point readPoint() throws IOException {
+    private Point readPoint() throws IOException, ParseException {
```

Comment:
Too bad they added a new exception in a method signature in a bugfix release, but as long as we don't backport this change, I see no issues, for GeoTools it should be fine. Maybe it should be reported in the upgrade notes?

Reply:
backporting of the 1.18.0 upgrade was blocked already due to api changes, so backporting this 1.18.1 upgrade is a nog go